modules/openapi-generator/src/main/resources/typescript-rxjs/runtime.mustache

```diff
@@ -3,8 +3,9 @@
 import { Observable, of{{#withProgressSubscriber}}, Subscriber{{/withProgressSubscriber}} } from 'rxjs';
 import { ajax, AjaxRequest, AjaxResponse } from 'rxjs/ajax';
 import { map, concatMap } from 'rxjs/operators';
+import { servers } from './servers';
 
-export const BASE_PATH = '{{{basePath}}}'.replace(/\/+$/, '');
+export const BASE_PATH = servers[0].getUrl();
```

Comment:
will there always be servers?
according to https://swagger.io/specification/#fixed-fields
`servers` is 
```
An array of Server Objects, which provide connectivity information to a target server. If the servers property is not provided, or is an empty array, the default value would be a Server Object with a url value of /.
```
Did you check this works if we have an empty list?

Reply:
Yes, we do the same. If no server is provided, we will get:
```
const server1 = new ServerConfiguration<{  }>("/", {  }, "")

export const servers = [server1];
```

Regarding the previous replace statement -> we could end up with //endpoint
Where we have two / following each other. In principle this is not expected, but normally works.

The experimental typescript generator would have double slashes as well. What is your take on it? Shall we make sure we have no trailing /?

Reply:
good point, but I guess the double slash issue is already present with the current implementation. Maybe you could open an issue for that and handle this separately? so we could merge this PR

Reply:
Agreed, lets merge.

I will check if I have a minute or two the next days, to open a PR...I guess it is anyway not so urgent